kernel-install/50-mkosi.install

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1+
+
+import argparse
+import logging
+import os
+import shutil
+from pathlib import Path
+from typing import NamedTuple, Optional
+
+from mkosi.config import OutputFormat, __version__
+from mkosi.log import die, log_setup
+from mkosi.run import run, uncaught_exception_handler
+from mkosi.types import PathString
+
+
+class Context(NamedTuple):
+    command: str
+    kernel_version: str
+    entry_dir: Path
+    kernel_image: Path
```

Comment:
No idea why mypy doesn't see this, but since the argparse default is `None` and `entry_dir` and `kernel_image` are `?`, in most cases `None` will be passed into the `Context` constructor further down. 
```suggestion
    entry_dir: Optional[Path]
    kernel_image: Optional[Path]
```

Reply:
I made these required instead as they're documented to always be passed by kernel-install